builtin/logical/aws/path_roles.go

```diff
@@ -521,6 +534,7 @@ type awsRoleEntry struct {
 	MaxSTSTTL                time.Duration     `json:"max_sts_ttl"`                           // Max allowed TTL for STS credentials
 	UserPath                 string            `json:"user_path"`                             // The path for the IAM user when using "iam_user" credential type
 	PermissionsBoundaryARN   string            `json:"permissions_boundary_arn"`              // ARN of an IAM policy to attach as a permissions boundary
+	SerialNumber             string            `json:"mfa_serial_number"`                     // Serial number or ARN fo the MFA device
```

Comment:
```suggestion
	SerialNumber             string            `json:"mfa_serial_number"`                     // Serial number or ARN of the MFA device
```

Wonder if it's also worth mentioning that it can be an ARN in the `framework.Path` fields description?